index.html

```diff
@@ -41,6 +43,191 @@ <h1 class="section-1-heading-mobile playfair-display-font font-color">Losalini R
         <!--Image in Section 1/Headline Section-->
         <div class="section-1-img-mobile"></div>
       </div>
+
+      <!--Section 2 | Projects Section for mobile screens-->
+      <section class="section-2-mobile">
+        <h2 class="section-2-heading-mobile playfair-display-font font-color">Projects</h2>
+
+        <!--Projects container for project cards-->
+        <div class="projects-container-mobile">
+          <!--Project Card 1-->
+          <div class="card-1-mobile">
+            <img src="images/image3.jpg" alt="Hands on a macbook keyboard" class="card-1-img-mobile"/>
+
+            <!--Project description-->
+            <div class="project-description bg-color">
+              <h3 class="poppins-font font-white font-weight-lighter">Project name goes here</h3>
+
+              <ul class="project-technologies font-white">
+                <li class="font-color mr-3">HTML/CSS</li>
+                <li class="font-color mr-3">Ruby on Rails</li>
+                <li class="font-color">JavaScript</li>
+              </ul>
+
+              <div class="see-this-project-btn-mobile">
+                <a href="#" class="poppins-font bg-color font-white">See this project <span class="material-icons ">arrow_forward</span></a>

```

Comment:
Hey @chelmerrox, when hovering over the button, it shows different background for text and the rest of the button. Kindly fix this bug. Also, the button should have some interaction when it is pressed. I would recommend adding it. The same goes for the rest of the buttons in the Project section.
![errorhover](https://user-images.githubusercontent.com/33268927/170097674-e8ebd9e7-e020-4a98-ade2-03e9de397452.png)